tests/unit/remote/gdrive/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from dvc.repo import Repo
+from dvc.remote.gdrive import RemoteGDrive
+
+
+@pytest.fixture()
+def repo():
+    return Repo(".")
```

Comment:
You are using repo of the dvc git project itself, let's not do that. If you need Repo instance, you can use `dvc_repo` fixture that we have already.